api/src/main/java/com/github/eyefloaters/console/api/ClientFactory.java

```diff
@@ -47,11 +42,8 @@
 @ApplicationScoped
 public class ClientFactory {
```

Comment:
Can we have a sentence of javadoc explaining what the responsibilities of this class are?

Reply:
JavaDoc added for `ClientFactory`